src/java/org/apache/cassandra/service/accord/AccordService.java

```diff
@@ -243,18 +243,23 @@ public synchronized static void startup(NodeId tcmId)
         }
         instance = as;
 
-        replayJournal(as);
+        // If we hit an error during journal replay, we need to mark ourselves unsafe to read, perform full data repair,
+        // and trigger RX before we can continue serving traffic
+        if (replayJournal(as) && ClusterMetadata.current().directory.allJoinedEndpoints().size() > 1)
```

Comment:
is allJoinedEndpoints.size() > 1 enough? Don't we need a quorum (or some stronger user command insisting we proceed with a minority)?

Reply:
Oh, we definitely need that in order to succeed. Here, I am just checking for whether or not we are in the single node cluster, in which case rebootstrap will simply not work.